Sources/PayPalMessages/Enums/Environment.swift

```diff
@@ -78,26 +71,55 @@ public enum Environment: Equatable {
         case log = "/v1/credit/upstream-messaging-events"
     }
 
-    func url(_ path: PayPalMessagePath, _ queryParams: [String: String?]? = nil) -> URL? {
-        var parts = URLComponents()
-        var queryItems: [URLQueryItem]?
+    private var devTouchpoint: Bool {
+        switch self {
+        case .stage(_, let devTouchpoint, _):
+            return devTouchpoint
+        default:
+            return false
+        }
+    }
 
-        if let queryParams, !queryParams.isEmpty {
-            queryItems = queryParams.map { URLQueryItem(name: $0.key, value: $0.value) }
+    private var stageTag: String? {
+        switch self {
+        case .stage(_, _, let stageTag):
+            return stageTag
+        default:
+            return nil
         }
+    }
```

Comment:
Are these being used anywhere since they are `private`? I think they may be leftover from previous changes. Are they safe to remove?